### PR TITLE
Update Products & Startups in README.md

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to add links in the Update Products & Startups section in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +79,8 @@ A list of of AI coding topics.
 - [Safurai](https://www.safurai.com/)
 - [phind](https://www.phind.com/)
 - [AskCodi](https://www.askcodi.com/)
+- [MutableAI](https://mutable.ai/)
+- [Refact](https://refact.ai/)
```
Comment (wsxiaoys) [inline README.md:83]: Mentioned in line 70

File: README.md [hunk 2]
```diff
@@ -60,8 +60,12 @@ A list of of AI coding topics.
 
 ## Products & Startups
 
-- [Copilot](https://github.com/features/copilot) + [Copilot X](https://github.com/features/preview/copilot-x)
+### Test Case Generation
 - [Codeium](https://www.codeium.com/)
```
Comment (wsxiaoys) [inline README.md:64]: Codeium is not for test case generation, guessing you mean codium ?

File: README.md [hunk 3]
```diff
@@ -60,8 +60,12 @@ A list of of AI coding topics.
 
 ## Products & Startups
 
-- [Copilot](https://github.com/features/copilot) + [Copilot X](https://github.com/features/preview/copilot-x)
+### Test Case Generation
```
Comment (wsxiaoys) [inline README.md:63]: Prefer not to split the startup into sub categories - as they pivot time to time.

File: README.md [hunk 4]
```diff
@@ -60,8 +60,12 @@ A list of of AI coding topics.
...
 - [Codeium](https://www.codeium.com/)
+- [Machinet](https://machinet.net/)
```
Comment (wsxiaoys) [inline README.md:65]: Please append Machinet to end of startup list.

**Review by wsxiaoys — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (leemgs) [reply README.md:64]: Yes.

File: README.md (hunk 3 above)
Comment (leemgs) [reply README.md:63]: Got it. Updated.

File: README.md (hunk 4 above)
Comment (leemgs) [reply README.md:65]: Got it. Updated.

**Comment by leemgs** [comment]
Updated. Thanks.

**Pull request merged**